<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptWindow</class>
 <widget class="QMainWindow" name="EncryptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncryptWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_INPUT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_OUTPUT">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_INPUT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_OUTPUT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_DISPLAY">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>911</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SOMETHING HAPPANED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_NEW">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>NEW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SAVE">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LOAD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ENCODE">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ENCODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DECODE">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DECODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CLEAR">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_COPY">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>COPY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
